teku/src/main/java/tech/pegasys/teku/cli/options/DataOptions.java

```diff
@@ -48,9 +48,9 @@
       paramLabel = "<VERSION>",
       description = "Database version to create (3 or 4)",
```

Comment:
```suggestion
      description = "Database version to create (3.0 or 4)",
```

Although potentially we should just make this a `DatabaseVersion` type?  Having to specify `V3` or `V4` isn't ideal but for a hidden option it would be fine and makes sure we always have a valid value.